experimental/devices/sn3218/example_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package sn3218_test
+
+import (
+	"log"
+	"periph.io/x/periph/conn/i2c/i2creg"
+	"periph.io/x/periph/experimental/devices/sn3218"
+	"periph.io/x/periph/host"
+	"time"
+)
+
+func Example() {
+	if _, err := host.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	b, err := i2creg.Open("")
+
+	defer b.Close()
```

Comment:
This ordering would panic if err != nil since b == nil

Reply:
fixed